### ISSUE TITLE
Live search is not visible

### ISSUE THREAD

**Issue by garak** [doctrine/doctrine-website#258]
It looks like the live search on Doctrine docs is visible only if you don't scroll page.
Example: you open a page, you search in the top bar: it works, you see results.
Negative example: you open a page, you scroll down, then you search in the top bar. It does not work as intended, since results are not placed in absolute way, so you can't see them unless you scroll back to top

**Comment by SenseException** [comment]
Yes, you have to scroll up to see the search results. This is currently a workaround to make the search more usable for bigger results, that is cut by some browsers where the bottom results can't be reached by scrolling anymore. The same behavior also made mobiles less usable.

As you correctly state, the search still needs to be improved in usability and result relevance.
